grouper/ctl/dump_sql.py

```diff
@@ -0,0 +1,25 @@
+from typing import TYPE_CHECKING
+
+from sqlalchemy.schema import CreateTable
+
+from grouper.models.base.model_base import Model
+from grouper.models.base.session import get_db_engine
+from grouper.settings import settings
+from grouper.util import get_database_url
+
+if TYPE_CHECKING:
+    import argparse  # noqa
+
+
+def dump_sql_command(args):
+    # type: (argparse.Namespace) -> None
+    db_engine = get_db_engine(get_database_url(settings))
+    for table in Model.metadata.sorted_tables:
+        print CreateTable(table).compile(db_engine)
+
+
+def add_parser(subparsers):
+    # type: (argparse._SubParsersAction) -> None
+    print type(subparsers)
```

Comment:
Looks like this got left behind, causing the test failure

Reply:
Whoops, thank you!  Finished this relatively late last night and didn't check.